apps/frontend/src/components/filters.tsx

```diff
@@ -0,0 +1,250 @@
+      latitude: 48.463302,
+      longitude: -123.3108,
+    });
+  };
+
+  // Gets the user location, returns false on failure, defaults to uvic
+  const getLocation = () => {
+    try {
+      navigator.geolocation.getCurrentPosition((position) => {
+        const currentLatitude = position.coords.latitude;
+        const currentLongitude = position.coords.longitude;
+        if (currentLatitude !== 0 && currentLongitude !== 0) {
+          setLatitude(currentLatitude);
+          setLongitude(currentLongitude);
+          handleApplyFilters();
+          return true;
+        }
+      });
+      return false;
+    } catch (error) {
+      return false;
+    }
+  };
+
+  useEffect(() => {
+    setMinPrice(filters.minPrice);
+    setMaxPrice(filters.maxPrice);
+    setStatus(filters.status);
+    setType(filters.searchType);
+    setLatitude(filters.latitude);
+    setLongitude(filters.longitude);
+  }, [filters]);
+
+  const isDesktop = useMediaQuery("(min-width:850px)");
+
+  return (
+    <Grid item container xs={isDesktop ? 5 : 10}>
+      <Typography variant="h6" component="h3" m={1}>
+        Filters
+      </Typography>
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "center",
+          justifyContent: "center",
+          width: "100%",
+        }}
+      >
+        <FormControl fullWidth sx={{ m: 1 }}>
+          <FormHelperText>Min</FormHelperText>
+          <OutlinedInput
+            id="outlined-adornment-amount"
+            startAdornment={<InputAdornment position="start">$</InputAdornment>}
+            placeholder="Min"
+            value={minPrice !== null ? minPrice : ""}
+            onChange={handleMinPriceChange}
+            type="number"
+          />
+        </FormControl>
+        <FormControl fullWidth sx={{ m: 1 }}>
+          <FormHelperText>Max</FormHelperText>
+          <OutlinedInput
+            id="outlined-adornment-amount"
+            startAdornment={<InputAdornment position="start">$</InputAdornment>}
+            placeholder="Max"
+            value={maxPrice !== null ? maxPrice : ""}
+            onChange={handleMaxPriceChange}
+            type="number"
+          />
+        </FormControl>
+      </Box>
+      <Box
+        sx={{
+          display: "flex",
+          flexDirection: "row",
+          alignItems: "center",
+          justifyContent: "center",
+          width: "100%",
+        }}
+      >
+        <FormControl fullWidth sx={{ m: 1 }}>
+          <FormHelperText>Status</FormHelperText>
+          <Select
+            value={status}
+            onChange={handleStatusChange}
+            displayEmpty
+            id="status-select"
+          >
+            <MenuItem disabled value="">
+              <em>Status</em>
+            </MenuItem>
+            <MenuItem value={"AVAILABLE"}>Available</MenuItem>
+            <MenuItem value={"SOLD"}>Not Available</MenuItem>
+          </Select>
+        </FormControl>
```

Comment:
I think it would be best if this defaulted to "AVAILABLE" in order to closer match the [OpenAPI spec](https://market.lebl.ca/openapi/#/Search%20and%20Recommendations/search) just like how you handled listing type.

|Listing Type|Listing Status|
|---|---|
| ![image](https://github.com/UVicMartletplace/martletplace/assets/54737724/e232401b-5ee0-4c36-a054-19e103026383) |![image](https://github.com/UVicMartletplace/martletplace/assets/54737724/6faf87bb-0e7d-4718-bbb2-18582f19f3f8) |